.circleci/config.yml

```diff
@@ -20,9 +20,36 @@ jobs:
       - run: npm install
       - run: npm test
 
+  deploy:
+    docker:
+      - image: circleci/node:8
```

Comment:
does it really has to be circleci node ?

Reply:
Not really.
Technically it could be any docker image that has got `npm`; however I decided to reuse the same because

1. As it has been already downloaded and built, it should start almost immediately.
2. The `deploy` job has a transitive dependency on node8 build — therefore it's kind of a natural dependency.

If you have another idea around that feel free to shout them!

Reply:
not only npm, but also tools to build `bcrypt` lib. As I remember `node:alpine` will work for that

and alpine itself is extremely small,  so push\pull time are reduced significantly   
if it works I would actually use it